include/SKR_mini_E3-2.0/Pins.h

```diff
@@ -116,14 +120,14 @@
 #define DSP_DC_PIN          -1
 #define DSP_RESET_PIN       -1
 
-#if !defined(USE_LEONERD_DISPLAY)
+#if defined(USE_LEONERD_DISPLAY)
```

Comment:
What's the improvement here?

Reply:
None, clearer that is removed debug_pin

Reply:
Only if you stop reading the file in the middle. Please remove that change.